### PR TITLE
Created A Make Workflow

### PR DESCRIPTION
**Pull request by nishantc1527**
Just to check if pull requests are valid.

### CODE REVIEW

**Comment by nishantc1527** [comment]
I have no idea why it failed since I just added one file, which is the workflow.

File: .github/workflows/c-cpp.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: configure
+      run: ./configure
```
Comment (htsign) [inline .github/workflows/c-cpp.yml:17]: I think no need to run `./configure`

**Comment by nishantc1527** [comment]
Done.

**Comment by nishantc1527** [comment]
Let me make another pull request so it get's rid of the ./configure.

**Pull request closed without merging**